subsys/CMakeLists.txt

```diff
@@ -13,6 +13,12 @@ add_subdirectory_ifdef(CONFIG_SECURE_BOOT_CRYPTO bootloader/bl_crypto)
 add_subdirectory_ifdef(CONFIG_SECURE_BOOT_VALIDATION bootloader/bl_validation)
 add_subdirectory_ifdef(CONFIG_SECURE_BOOT_STORAGE bootloader/bl_storage)
 
+if (CONFIG_SECURE_BOOT_STORAGE)
+  add_subdirectory(bootloader/bl_otp_counters)
```

Comment:
Let me clarify what you mean, does this mean that I can automatically include this from the bl_storage directory instead of having this cmake if statement here?